src/flutter/shell/platform/linux_embedded/window/linuxes_window_eglstream.cc

```diff
@@ -0,0 +1,444 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
```

Comment:
You need to consider to make it common with gbm.

Reply:
I created `xxx_drm.h` files for the common code of `drm` and `eglstream`, and created `xxx_drm_gbm.h` and `xxx_drm_eglstream.h` for the specific code of `drm` and `eglstream`.